mcp-servers/helm/templates/pvc.yaml

```diff
@@ -1,7 +1,7 @@
 {{- $root := . }}
 {{- $servers := include "mcp-servers.mergeMcpServers" . | fromJson }}
 {{- range $key, $server := $servers }}
-{{- if and $server.persistence $server.persistence.enabled }}
+{{- if and $server.mcpserver $server.mcpserver.enabled $server.persistence $server.persistence.enabled }}
```

Comment:
@rhkp @lokeshrangineni doesn't toolhive support the creation of a pvc? If so, we can remove this file entirely, and let it handle the storage.